encrypted-media-respec.html

```diff
@@ -3868,23 +3870,48 @@
             <li><p>If Key System messages contain information derived from a user identifier in a consistent manner, for example such that a portion of the initial Key System message for a specific content item does not change over time and is dependent on the user identifier, then this information could be used by any application to track the device or user over time.</p></li>
           </ul>
 
-          <p>In addition, if a Key System permits keys to be stored and to be re-used between origins, then it may be possible for two origins to collude and track a unique user by recording their ability to access a common key.</p>
+          <p>In addition, if a Key System permits keys or other data to be stored and to be re-used between origins, then it may be possible for two origins to collude and track a unique user by recording their ability to access a common key.</p>
           <p>Finally, if any user interface for user control of Key Systems presents data separately from data in HTTP session cookies [[COOKIES]] or persistent storage, then users are likely to modify site authorization or delete data in one and not the others.
             This would allow sites to use the various features as redundant backup for each other, defeating a user's attempts to protect his or her privacy.
           </p>
+          
+          <p>
+            In addition to the potential for sites and other third-parties to track users, the user agent and/or CDM implementer could build a profile of the user's activities or interests, such as sites using the APIs defined in this specification that the user visits.
+            Such tracking would undermine the privacy protections provided by the rest of the web platform, especially those related to isolation of origins.
```

Comment:
>The ability of the user agent to build a profile of the users' activities is not modified by this specification.

Agreed. I was trying to be inclusive since the user agent vendor could be the one receiving individualization requests. For the user agent vendor, it the difference would be addition of _remote_ tracking. In contrast, local tracking by a user agent would be very concerning and more obvious.

I think this text is fine, especially given the context, but I'm open to suggestions.